generator/src/test/java/com/scottlogic/deg/generator/cucumber/features/operators/grammatical/If.feature

```diff
@@ -2482,21 +2480,19 @@ Feature: Values can be specified by using if, then and else constraints
       """
       {
         "if": { "field": "foo", "is": "equalTo", "value": 1 },
-        "then": { "field": "bar", "is": "equalTo", "value": 1 },
-        "else": { "field": "bar", "is": "granularTo", "value": 0.01 }
+        "then": { "field": "bar", "is": "equalTo", "value": 3.33 },
```

Comment:
I guess you changed this value from `1` to `3.33` because you removed `1` from the set. What was the reason to remove it from the set, did you need to do this to get the tests to pass?

Reply:
I did this as 1 would be included in all granularities so it would be more difficult to tell if the rule was being followed